src/actions/index.js

```diff
@@ -10,9 +10,9 @@ export const DECREMENT = 'DECREMENT';
 // is left to the reducer(s).
 
 export const increment = () => {
-  // Fill in this function
+  return {type: INCREMENT}
```

Comment:
Good job here! You could have added a payload, but you recognized that you didn't need one.